ivadomed/config/config.json

```diff
@@ -1,22 +1,22 @@
 {
     "command": "train",
-    "gpu": 7,
+    "gpu": 0,
     "log_directory": "spineGeneric",
```

Comment:
how about something more explicit, like "training_results"?

Reply:
This folder contains: trained model, tensorboard check points (loss and metrics), the configuration file, the dataset split (image paths for training/V/Tessting), but also the nifti files reconstructed from the inference time (testing) and the related csv with metrics for each image.
So this is more a "experiment_directory" than "training_results per se. Would you prefer "experiment_directory" than "log_directory" @jcohenadad ?

Reply:
Hum, i see. Fundamentally I would not have a field here, but rather specify output results as a flag when calling ivadomed.

Reply:
Open issue #355